jobs/ServiceInstanceUpdateJob.js

```diff
@@ -39,25 +40,23 @@ class ServiceInstanceUpdateJob extends BaseJob {
         logger.error(msg);
         return this.runFailed(new errors.ServiceUnavailable(msg), operationResponse, job, done);
       }
-      return this
-        .getServicePlanIdForInstanceId(instanceDetails.instance_id)
-        .then((planId) => (planId === undefined ?
+      return eventmesh.apiServerClient.getResource({
+          resourceGroup: CONST.APISERVER.RESOURCE_GROUPS.DEPLOYMENT,
+          resourceType: CONST.APISERVER.RESOURCE_TYPES.DIRECTOR,
+          resourceId: instanceDetails.instance_id
+        })
+        .catch(errors.NotFound, () => undefined)
+        .then(resource => _.get(resource, 'spec.options'))
+        .then((resourceDetails) => (resourceDetails === undefined ?
```

Comment:
Ideally all the resource would be created once a scheduled update is triggered on live, still deletion is little risky, can we log erro to start with?

Reply:
as discussed leaving, as would be taken care by update.